cranelift/codegen/src/isa/x64/lower.isle

```diff
@@ -644,8 +736,10 @@
       (let ((amt_ Reg (lo_reg amt)))
         (value_reg (m_rotl ty (put_in_reg src) (Imm8Reg.Reg amt_)))))
 
-(rule (lower (has_type (ty_32_or_64 ty) (rotl src (imm8_from_value amt))))
-      (value_reg (m_rotl ty (put_in_reg src) amt)))
+(rule (lower (has_type (ty_32_or_64 ty) (rotl src (const_value amt))))
```

Comment:
IIRC we there's usage of `(def_inst (iconst N))` in other places, should that be used here or should other places be switched to this?